db/migrate/20140615175553_init_schema.rb

```diff
@@ -0,0 +1,871 @@
+    end
+
+    add_index 'c_edges', ['parent_id', 'child_id'], name: 'index_c_edges_on_parent_id_and_child_id', unique: true
+
+    create_table 'categories', force: true do |t|
+      t.text     'name',       null: false
+      t.string   'ordinal'
+      t.datetime 'created_at', null: false
+      t.datetime 'updated_at', null: false
+    end
+
+    create_table 'categories_ontologies', force: true do |t|
+      t.integer 'ontology_id', null: false
+      t.integer 'category_id', null: false
+    end
+
+    add_index 'categories_ontologies', ['category_id', 'ontology_id'], name: 'index_c_vertices_ontologies_on_c_vertex_id_and_ontology_id', unique: true
+    add_index 'categories_ontologies', ['ontology_id', 'category_id'], name: 'index_c_vertices_ontologies_on_ontology_id_and_c_vertex_id'
+
+    create_table 'code_references', force: true do |t|
+      t.integer  'begin_line'
+      t.integer  'end_line'
+      t.integer  'begin_column'
+      t.integer  'end_column'
+      t.integer  'referencee_id'
+      t.string   'referencee_type'
+      t.datetime 'created_at',      null: false
+      t.datetime 'updated_at',      null: false
+    end
+
+    create_table 'comments', force: true do |t|
+      t.integer  'commentable_id',   null: false
+      t.string   'commentable_type', null: false
+      t.integer  'user_id',          null: false
+      t.text     'text',             null: false
+      t.datetime 'created_at',       null: false
+      t.datetime 'updated_at',       null: false
+    end
+
+    add_index 'comments', ['commentable_id', 'commentable_type', 'id'], name: 'index_comments_on_commentable_and_id'
+    add_index 'comments', ['user_id'], name: 'index_comments_on_user_id'
+
+    create_table 'e_edges', force: true do |t|
+      t.integer  'parent_id'
+      t.integer  'child_id'
+      t.datetime 'created_at', null: false
+      t.datetime 'updated_at', null: false
+    end
+
+    add_index 'e_edges', ['child_id'], name: 'index_e_edges_on_child_id'
+    add_index 'e_edges', ['parent_id'], name: 'index_e_edges_on_parent_id'
+
+    create_table 'entities', force: true do |t|
+      t.integer  'ontology_id',                    null: false
+      t.string   'kind'
+      t.text     'text',                           null: false
+      t.text     'name',                           null: false
+      t.text     'iri'
+      t.string   'range'
+      t.integer  'comments_count',  default: 0, null: false
+      t.datetime 'created_at',                     null: false
+      t.datetime 'updated_at',                     null: false
+      t.text     'display_name'
+      t.text     'label'
+      t.text     'comment'
+      t.integer  'entity_group_id'
+    end
+
+    add_index 'entities', ['display_name'], name: 'index_entities_on_display_name'
+    add_index 'entities', ['name'], name: 'index_entities_on_name'
+    add_index 'entities', ['ontology_id', 'id'], name: 'index_entities_on_ontology_id_and_id', unique: true
```

Comment:
Line is too long. [107/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.